tasks/xpack/security/elasticsearch-security.yml

```diff
@@ -28,12 +28,62 @@
 
   - name: Create Bootstrap password for elastic user
     become: yes
-    shell: echo {{ es_api_basic_auth_password | quote }} | {{ es_home }}/bin/elasticsearch-keystore add -x 'bootstrap.password'
+    command:
```

Comment:
CI tests using this task are failing (example: https://devops-ci.elastic.co/job/elastic+ansible-elasticsearch+pull-request/237/OS=ubuntu-1404,TEST_TYPE=xpack-upgrade,label=linux/console).

I did a few local tests and see 2 issues:

- `argv` syntax for `command` module was released in Ansible 2.6 but our role is still supporting Ansible 2.5

  https://github.com/elastic/ansible-elasticsearch/blob/87946a8305147216cc682df35054f5e9b75ae953/meta/main.yml#L10

  I would prefer not remove 2.5 support if possible as it is the default version used in Ubuntu 18.04 packages and in our `ubuntu-1804` tests.

- The generated command don't seem valid. When using it with a more recent Ansible version and enabling log for this task, I get the following error:

  ```
  TASK [elasticsearch : Create Bootstrap password for elastic user] **************
  fatal: [localhost]: FAILED! => {"changed": false, "cmd": "changeme /usr/share/elasticsearch/bin/elasticsearch-keystore add -x bootstrap.password", "msg": "[Errno 2] No such file or directory", "rc": 2}
  ```

Reply:
Ok, I can have a look at this and change it back. How should we address the problem mentioned by @schalle about autogenerated passwords?

Reply:
AFAIK the Ansible quote filter should handle the special characters correctly so that we don't have any issue.

I did a try with a few passwords like `elastic|$&01` and didn't have any issue with my small samples.

Reply:
Ok. Got it. I have fixed it and tried to push it but used a wrong git user. Can you delete this pull request please, I will do it again

Reply:
@jmlrt Or even better, just accept it.

Reply:
Not sure that we can merge the PR with CLA failing on some commits.

Can you try changing the author name of these 2 commits with [this procedure](https://stackoverflow.com/a/3042512) for example?

Reply:
@jmlrt I have tried it. Can you please check? Thanks

Reply:
This is still failing and I still see your other author name and emails in some commits. Then if you prefer creating a new branch/PR please do, you can then close this one or I'll do it when the new one will be created.